runtime/polkadot/src/lib.rs

```diff
@@ -2125,7 +2125,9 @@ mod test_fees {
 			"can support {} voters in a single block for council elections; total bond {}",
 			voters, cost_dollars,
 		);
-		assert!(cost_dollars > 150_000); // DOLLAR ~ new DOT ~ 10e10
+		// The minimal number of voters we expect per block.
+		assert!(voters >= 1_000);
+		assert!(cost_dollars >= 10_000);
```

Comment:
It changed from 150k to 126k because the weights are updated. Now instead of slightly adjusting it I reduced the values so that they wont error again in benchmarking, but still sanity-check.  
(btw this is the 0.9.26 branch update, not 0.9.27)